### ISSUE TITLE
cataract

### ISSUE THREAD

**Issue by xsnajperx** [obophenotype/human-phenotype-ontology#10237]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by xsnajperx** [comment]
Thanks
I is my very begging with HP
Thank You one more.

W dniu śr., 22.11.2023 o 16:35 jagcgc ***@***.***> napisał(a):

**Comment by jagcgc** [comment]
No problem! Glad to be of help @xsnajperx

**Comment by pnrobinson** [comment]
thanks
